onebusaway-android/src/main/java/org/onebusaway/android/directions/util/ConversionUtils.java

```diff
@@ -0,0 +1,464 @@
+/*
+ * Copyright 2012 University of South Florida
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and 
+ * limitations under the License.
+ */
+
+package org.onebusaway.android.directions.util;
+
+import android.content.Context;
+import android.preference.PreferenceManager;
+import android.text.SpannableString;
+import android.text.TextUtils;
+import android.text.style.ForegroundColorSpan;
+import android.text.style.StrikethroughSpan;
+
+import org.onebusaway.android.R;
+import org.onebusaway.android.util.RegionUtils;
+import org.opentripplanner.api.model.Itinerary;
+import org.opentripplanner.api.model.Leg;
+import org.opentripplanner.routing.core.TraverseMode;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+
+
+/**
+ * @author Khoa Tran
+ * @author Simon Jacobs - conversion for OBA. Added Imperial measurements
+ */
+
+public class ConversionUtils {
+
+    public static double getDuration(String startTimeText, String endTimeText,
+                                     Context applicationContext) {

```

Comment:
Could you please add better Javadocs to these methods?  Should at a minimum include what the method does, along with `@param` and `@return` for parameters and return variables, respectively.